dcos/dcos-ui
# DCOS-48532: No text button in Marathon configuration

Add a text for a textless button.

Closes https://jira.mesosphere.com/browse/DCOS-48532

## Testing
1. Go to catalog.
2. Open `marathon`.
3. Click "Review & Run".
4. Open the "Marathon" tab and scroll to the bottom.
5. Verify that the button is not empty.

## Trade-offs
Somewhat of a band-aid solution.

## Dependencies
None.

## Screenshots
### Before
![Снимка от 2019-06-12 15-59-26](https://user-images.githubusercontent.com/40791275/59353187-1364e400-8d2b-11e9-918a-0201e497f695.png)

### After
![Снимка от 2019-06-12 15-38-21](https://user-images.githubusercontent.com/40791275/59353055-cbde5800-8d2a-11e9-85da-fce951c67937.png)
